static/js/utils/chart_csv_utils.ts

```diff
@@ -191,8 +195,8 @@ export function mapDataToCsv(layerData: MapLayerData[]): string {
     }
   });
   const header = hasVariable
-    ? ["label", "variable", "data"]
-    : ["label", "data"];
+    ? ["place", "variable", "data"]
```

Comment:
Are we not gating this on TOOLFORMER_RAG?

Reply:
i dont think this needs to be gated because map tiles in nodejs server is the only time this function is used & we don't return map tiles for bard